crates/agent/src/publications/handler.rs

```diff
@@ -35,19 +35,34 @@ impl Handler for Publisher {
                 .context("clearing old errors")?;
 
             let time_queued = chrono::Utc::now().signed_duration_since(row.updated_at);
-            let result = self.process(row).await?;
 
-            draft::insert_errors(draft_id, result.draft_errors(), &mut txn).await?;
+            let (status, draft_errors) = match self.process(row).await {
+                Ok(result) => {
+                    let errors = result.draft_errors();
+                    (result.status, errors)
+                }
+                Err(error) => {
+                    tracing::warn!(?error, pub_id = %id, "build finished with error");
+                    let errors = vec![draft::Error {
+                        catalog_name: String::new(),
```

Comment:
is it okay with the UI if the draft_error `scope` column is completely empty?

Reply:
Great question. It took me a bit to test this because apparently postgresT doesn't allow you to insert specs containing null bytes. I was actually pretty happy to find that out. I wasn't able to draft anything including a null byte either using the advanced spec editor or via flowctl.